.github/workflows/check-merge-conflict.yml

```diff
@@ -0,0 +1,16 @@
+name: Merge Conflict Check
+on:
+  push:
+  pull_request_target:
+    types: [synchronize]
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: check if prs are dirty
+        uses: eps1lon/actions-label-merge-conflict@releases/2.x
+        with:
+          dirtyLabel: has-conflicts
+          repoToken: "${{ secrets.GITHUB_TOKEN }}"
+          commentOnDirty: "This pull request has conflicts, please resolve those before we can evaluate the pull request."
```

Comment:
Should we also `commentOnClean`? Or do we notice that by the removal of the label, alone? Are there notifications for label events? :thinking: Do we even want that?! :scream: 

No hard opinions here, just thinking about how we will use this.

Reply:
Almost always, the conflicts will be resolved by the author pushing to the branch. So the author is already active at that time (unlike when the conflicts arise). So I think it's fine not to comment and not to notify.

Reply:
The notifications would be for potential reviewers in that case, though.

Reply:
Reviewers get notified when someone pushes on the branch, right?

Reply:
Yes, they would.
I also don't think adding more notifications is really helpful when we know that they are almost universally not helpful.
We would just start ignoring them

Reply:
Fair enough.